winterfell/src/lib.rs

```diff
@@ -584,6 +584,7 @@
 //!     FieldExtension::None,
 //!     8,  // FRI folding factor
 //!     31, // FRI max remainder polynomial degree
+//!     BatchingMethod::Linear, // Type of batching used in computing DEEP polynomial
```

Comment:
nit: I would change the comment to "// method of batching used in computing DEEP polynomial".